pyproject.toml

```
@@ -51,7 +51,7 @@ documentation = "https://docs.saleor.io/"
   jaeger-client = "^4.5.0"
   openpyxl = "^3.0.3"
   django-cache-url = "^3.1.2"
-  pyjwt = ">=2.4.0,<3.0.0"
+  pyjwt = "2.5.0" # locked because version 2.6.0 changed iat validation that breaks Saleor tests, will be handled in seperate PR
```

A link to the issue to make it easier to find it
```suggestion
  pyjwt = "2.5.0" # Version 2.6.0 changed iat validation which causes tests to fail: https://github.com/saleor/saleor/issues/11047
```